### ISSUE TITLE
Support for lists?

### ISSUE THREAD

**Issue by htilly** [celiao/tmdbsimple#59]
Does it support get lists? For both Movies and TV Shows?
I.e. my saved watchlists?

https://developers.themoviedb.org/3/lists/get-list-details

**Comment by celiao** [comment]
tmdbsimple supports these movie lists:
- https://developers.themoviedb.org/3/movies/get-movie-lists

Would you like to contribute code and unit tests for get lists?
- https://developers.themoviedb.org/3/lists/get-list-details

**Comment by d762** [comment]
```
    discover = tmdb.Discover()
    kwargs = {'vote_count.gte': votecount, 'vote_average.gte': averagescore, 'release_date.gte': fromdate, 'with_genres': genres}
    response = discover.movie(**kwargs)
    if len(discover.results) > 0: 
        num = random.randint(0, len(discover.results))
        with open("~/list.txt", 'r') as f:
            content = f.readlines()
            content = [x.strip() for x in content]   
        if str(discover.results[num-1]['id']) in content: 
            getmovie()
        else:
            with open("~/list.txt", 'a') as f: f.write(str(discover.results[num-1]['id'])+"\n")
            print("Playing: " + str(discover.results[num-1]['title']))   
            return (discover.results[num-1]['id'])         
```

**Comment by celiao** [comment]
# List Example

`tmdbsimple` currently implements TMDB v3, where lists of movies are supported.  See https://developers.themoviedb.org/3/lists/get-list-details.

When TMDB v4 is implemented per #63, lists of both movies and TV series will be supported.  See https://developers.themoviedb.org/4/list/get-list.

Import tmdbsimple
----

```python
import tmdbsimple as tmdb
tmdb.API_KEY = '<your api key>'
SESSION_ID = '<your session id>'
```

Create List
----


```python
kwargs = {
            'name': 'My List',
            'description': 'My list of movies',
         }
lst = tmdb.Lists(session_id=SESSION_ID)
lst.list_create(**kwargs)
lst.info()
```




    {'created_by': 'celiao',
     'description': 'My list of movies',
     'favorite_count': 0,
     'id': '7054530',
     'items': [],
     'item_count': 0,
     'iso_639_1': 'en',
     'name': 'My List',
     'poster_path': None}



Add Movie to List
----


```python
THE_MATRIX = 603
lst.add_item(media_id=THE_MATRIX)
lst.info()
```




    {'created_by': 'celiao',
     'description': 'My list of movies',
     'favorite_count': 0,
     'id': '7054530',
     'items': [{'poster_path': '/f89U3ADr1oiB1s9GkdPOEpXUk5H.jpg',
       'popularity': 36.618,
       'vote_count': 17524,
       'video': False,
       'media_type': 'movie',
       'id': 603,
       'adult': False,
       'backdrop_path': '/ByDf0zjLSumz1MP1cDEo2JWVtU.jpg',
       'original_language': 'en',
       'original_title': 'The Matrix',
       'genre_ids': [28, 878],
       'title': 'The Matrix',
       'vote_average': 8.1,
       'overview': 'Set in the 22nd century, The Matrix tells the story of a computer hacker who joins a group of underground insurgents fighting the vast and powerful computers who now rule the earth.',
       'release_date': '1999-03-30'}],
     'item_count': 1,
     'iso_639_1': 'en',
     'name': 'My List',
     'poster_path': None}
